packages/commercetools/composables/src/useUser/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ref, Ref } from '@vue/composition-api'
+import { UseUser } from '@vue-storefront/interfaces'
+import { CustomerSignMeInDraft, CustomerSignMeUpDraft, Customer } from '@vue-storefront/commercetools-api/lib/src/types/GraphQL'
+import  { customerSignMeUp } from '@vue-storefront/commercetools-api'
+
+type UserRef = Ref<Customer>
+type RegisterFn = (userData: CustomerSignMeInDraft) => Promise<void>
```

Comment:
perhaps we could have a more generic type for that and transform to draft format inside composable?